src/_guides/language/language-tour.md

```diff
@@ -718,6 +718,19 @@ var list = [1, 2, 3];
   [type inference.](/guides/language/type-system#type-inference)
 {{site.alert.end}}
 
+Trailing commas are allowed in Dart collections. 
+They don't affect the collection, but having them can 
+prevent errors when copy-pasting code. 
```

Comment:
I'd like to clarify the meaning of "trailing comma" and avoid the ambiguous "they/them". What do you think of this?

```suggestion
You can add a comma after the last item in a Dart collection literal.
This _trailing comma_ doesn't affect the collection,
but it can help prevent copy-paste errors.
```

Reply:
#1404 mentions other places where trailing commas work, too. I started trying to update this to include those additional places. (Grepping for `','?` and `(',')?` in the spec, it looks like it's enums, args, params, & asserts — probably all the comma-delimited things in the language except perhaps initializers.) But it started getting too complicated. Maybe we should cover each of those cases where they come up?